NotesApi/Startup.cs

```diff
@@ -62,12 +64,11 @@ public void ConfigureServices(IServiceCollection services)
             services.AddCors();
 
             services
-                .AddMvc()
+                .AddControllers()
```

Comment:
That's curious. What's does this change?

Reply:
From [here](https://dotnettutorials.net/lesson/difference-between-addmvc-and-addmvccore-method/): 

The MVC model isn't needed for this API since it doesn't use views etc. So we're just paring down the extra unnecessary functionality and improving startup times.

![image](https://github.com/user-attachments/assets/6155fdde-6e8e-4dc8-92a8-4ada2f71a1c6)